src/Credentials/EcsCredentialProvider.php

```diff
@@ -97,13 +97,24 @@ public function __invoke()
     private function getEcsAuthToken()
     {
         if (!empty($path = getenv(self::ENV_AUTH_TOKEN_FILE))) {
-            if (is_readable($path)) {
-                return file_get_contents($path);
+            $token =  @file_get_contents($path);
+            if (false === $token) {
+                clearstatcache(true, dirname($path) . "/" . @readlink($path));
```

Comment:
instead of hard-coding a forward slash, could you replace it with `\DIRECTORY_SEPARATOR`?

Reply:
Sure. I think it is safer in that way. 

Thanks!